zerver/tests/test_messages.py

```diff
@@ -2808,7 +2809,8 @@ def test_propagate_all_topics(self) -> None:
         result = self.client_patch("/json/messages/" + str(id2), {
             'message_id': id2,
             'topic': 'edited',
-            'propagate_mode': 'change_all'
+            'propagate_mode': 'change_all',
+            'client_oldest_message_id': id1
         })
         self.assert_json_success(result)
 
```

Comment:
The tests should include something that actually tests the new corner cases (e.g. that if you set an a `client_oldest_message_id` that's not very old it doesn't edit older messages, edit)